scanner/e2etests/image_test.go

```diff
@@ -0,0 +1,244 @@
+package e2etests
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"os"
+	"strings"
+	"testing"
+
+	"github.com/google/go-containerregistry/pkg/authn"
+	"github.com/google/go-containerregistry/pkg/name"
+	v4 "github.com/stackrox/rox/generated/internalapi/scanner/v4"
+	"github.com/stackrox/rox/pkg/utils"
+	"github.com/stackrox/rox/scanner/client"
+	"github.com/stackrox/rox/scanner/indexer"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+// Vulnerability describes a vulnerability in a TestCase.
+type Vulnerability struct {
+	Name        string                 `json:"Name"`
+	Description string                 `json:"Description"`
+	Link        string                 `json:"Link"`
+	Severity    string                 `json:"Severity"`
+	Metadata    map[string]interface{} `json:"Metadata"`
+	FixedBy     string                 `json:"FixedBy"`
+}
+
+// Vulnerability describes a feature in a TestCase.
+type Feature struct {
```

Comment:
This is fine for now, but at some point I'd love to get rid of the old terminology. `Feature` is definitely a remnant of Scanner v2, which I'd like to rename at some point. Same with `Namespace`. Not going to block the PR over these, though

Reply:
We are using the old terminology to onboard more easily Scanner V2 tests. Moving this to a new terminology is better done in a different when Scanner V4 E2E becomes more independent.